src/com/puppetlabs/puppetdb/config.clj

```diff
@@ -320,25 +330,21 @@
                      (assoc :url-prefix url-prefix)
                      (utils/assoc-when :update-server "http://updates.puppetlabs.com/check-for-updates"))))))
 
-(defn warn-if-sslv3
-  "If the ssl-protocols config is present and contains sslv3, warn the user. Logging
-   at this point may or may not be setup, so warning via *err* and logging is done."
+(defn fix-tk-config
+  "Fix configuration before passing it to trapperkeeper.
+
+   In particular:
+   * Move certificate-whitelist from [jetty] to [global]"
   [config-data]
-  (when-let [protocol-str (get-in config-data [:jetty :ssl-protocols])]
-    (when (re-matches #"(?i).*sslv3.*" protocol-str)
+  (if-let [cw (get-in config-data [:jetty :certificate-whitelist])]
+    (do
+      ;; Log to stderr, logging is not yet initialized (and may never be).
       (binding [*out* *err*]
-        (let [warn-str "`ssl-protocols` contains SSLv3, a protocol with known vulnerabilities and should be removed from the `ssl-protocols` list"]
-          (println warn-str)
-          (log/warn warn-str)))))
-  config-data)
-
-(defn default-ssl-protocols
-  "Provide a default for ssl-protocols (that does NOT include sslv3). If sslv3
-   is present in the user provided config, warn the user."
-  [config-data]
-  (if (get-in config-data [:jetty :ssl-protocols])
-    (warn-if-sslv3 config-data)
-    (assoc-in config-data [:jetty :ssl-protocols] "TLSv1, TLSv1.1, TLSv1.2")))
+          (println "Option `certificate-whitelist` in [jetty] is now deprecated, the option must now be placed in [puppetdb]"))
```

Comment:
You can swap the binding for [this] (https://github.com/puppetlabs/puppetdb/blob/stable/src/com/puppetlabs/puppetdb/utils.clj#L32)